### PR TITLE
Named maps error responses with valid format

### LINKED ISSUE #305: Named maps errors should adhere to error specification
Some responses are not adhering to the [errors specification](https://github.com/CartoDB/Windshaft-cartodb/blob/2.4.1/docs/Map-API.md#errors) when they should.

`https://iriberri.cartodb.com/api/v1/map/named/world_borders_priv3/jsonp?callback=cb`

### Returns

```
cb({"error":"Error: Postgis Plugin: ERROR:  column \"usa\" does not exist\nLINE 1: ...FROM (select * from world_borders_10 WHERE name = USA) as cd...\n                                                             ^\n\nin executeQuery Full sql was: 'SELECT * FROM (select * from world_borders_10 WHERE name = USA) as cdbq LIMIT 0'\n  encountered during parsing of layer 'layer0' in Layer"});
```

### It should return

```
cb({"errors":["Error: Postgis Plugin: ERROR:  column \"usa\" does not exist\nLINE 1: ...FROM (select * from world_borders_10 WHERE name = USA) as cd...\n                                                             ^\n\nin executeQuery Full sql was: 'SELECT * FROM (select * from world_borders_10 WHERE name = USA) as cdbq LIMIT 0'\n  encountered during parsing of layer 'layer0' in Layer"]});
```

### PR DESCRIPTION
**Pull request by rochoa**
All error responses now returning with `{ "errors": ["message"] }` instead of `{ "error": "message" }`.

Fixes #305

### CODE REVIEW

**Comment by rochoa** [comment]
@alonsogarciapablo did you have the time to review this one? :smile:

File: test/acceptance/templates.js [hunk 1]
```diff
@@ -1131,10 +1131,11 @@ describe('template_api', function() {
           assert.equal(res.statusCode, 403,
             'Fetching tile with no auth: ' + res.statusCode + ': ' + res.body);
           var parsed = JSON.parse(res.body);
-          assert.ok(parsed.hasOwnProperty('error'),
-            "Missing 'error' from response body: " + res.body);
-          assert.ok(parsed.error.match(/permission denied/i),
-            'Unexpected error for unauthorized instance (expected /permission denied): ' + parsed.error);
+            console.log(parsed);
```
Comment (alonsogarciapablo) [inline test/acceptance/templates.js]: There's a log here.

**Comment by alonsogarciapablo** [comment]
LGTM :+1: Thanks!

**Comment by rochoa** [comment]
Removed the `console.log` and merging it. Thanks!

**Pull request merged**